chromadb/utils/embedding_functions/amazon_bedrock_embedding_function.py

```diff
@@ -1,29 +1,24 @@
+import json
 import logging
-
-from chromadb.api.types import (
-    Documents,
-    EmbeddingFunction,
-    Embeddings,
-)
-
 from typing import Any
-import json
 
-logger = logging.getLogger(__name__)
+from chromadb.api.types import Documents, EmbeddingFunction, Embeddings
 
+logger = logging.getLogger(__name__)
 
 
 class AmazonBedrockEmbeddingFunction(EmbeddingFunction[Documents]):
     def __init__(
         self,
-        session: "boto3.Session",  # noqa: F821 # Quote for forward reference
+        session: Any,
```

Comment:
It's not clear to me why `mypy` was complaining with the literal `boto3` so I decided to put an `Any`

Reply:
Types are really nice but both python and ts have a way to drive you into using `Any`. Maybe we can just try with `instanceof`.

I also like how the ecosystem is moving away from trying to have it all in the core package. Take for example how Langchain🦜🔗 has a partner lib for Chroma that is a completely separate package with all necessary deps. But that is probably a problem for another PR :)

Reply:
Wow, I really like [that approach](https://github.com/langchain-ai/langchain/tree/master/libs/partners/chroma), but agreed, maybe a problem for the future